emily_cron/app/services/deposit_processor.py

```diff
@@ -203,6 +204,49 @@ def process_expired_locktime(
 
         return updates
 
+    def process_long_pending(
+        self,
+        enriched_deposits: list[EnrichedDepositInfo],
+    ) -> list[DepositUpdate]:
+        """Process long-pending transactions.
+        Args:
+            enriched_deposits: List of enriched deposit information
+            stacks_chaintip: Current Stacks block info
+        Returns:
+            list[DepositUpdate]: List of deposit updates
+        """
+        updates = []
+
+        # Get the current time
+        current_time = int(datetime.now().timestamp())
+
+        long_pending_txs = [
+            tx
+            for tx in enriched_deposits
+            if tx.status == RequestStatus.PENDING.value  # Only check pending transactions
+            and not tx.in_mempool  # that we can't find via the mempool API (it might have been dropped)
+            and current_time - tx.deposit_time
+            >= settings.MAX_UNCONFIRMED_TIME  # and has been pending for too long
+        ]
```

Comment:
maybe nit: moving the last comment on its own line before avoiding breaking the last condition may improve the list comprehension readability